### PR TITLE
Updated Homework #6 for 2022

### PR DESCRIPTION
**Pull request by borjasotomayor**
This homework has undergone only minor updates, but some of the tasks had to be tweaked slightly to account for the fact that GitHub Actions no longer supports the `ubuntu-16.04` environment, and that the building instructions for Criterion have changed. Other than that, the tasks themselves are basically the same as last year.

### CODE REVIEW

File: website/hw/hw6_rubric.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  jobs from a matrix.
+
+ESNU Score
+----------
+
+The ESNU score will be determined as follows:
+
+- Excellent: No major issues, and at most one other issue.
+- Satisfactory: No major issues, and at most three other issues.
+- Needs Improvement: At least one major issue, or five other issues.
```
Comment (jacknugent1529) [inline website/hw/hw6_rubric.rst:47]: What would happen in the case of 4 minor issues?

**Review by jacknugent1529 — COMMENTED** [review]
I ran through the commands and everything worked as expected. My only other comment is that, based on hw1, it might be helpful to reiterate that citing sources means providing a link to the source. I think this would be helpful in the rubric or the hw itself.

File: website/hw/hw6_rubric.rst (hunk 1 above)
Comment (borjasotomayor) [reply website/hw/hw6_rubric.rst]: Oops, that should've said "or at least four other issues".

**Comment by borjasotomayor** [comment]
@jacknugent1529: I've pushed an update that reiterates that, for online sources, they should include the URL of the resource they consulted (and clarified that it should be specified in the text field for the relevant task on Gradescope). I've also fixed the rubric so the number of issues make sense.

**Review by ecoble — APPROVED** [review]
Everything looks good on the read through!

**Pull request merged**